### ISSUE TITLE
When use xcodebuild bunch of message: cannot merge previous GCDA file: corrupt arc tag

### ISSUE THREAD

**Issue by carya** [markohlebar/Peckham#69]
When I use xcodebuild to archive, I get a bunch of these messages:

profiling: xx/Alcatraz/Plug-ins/Peckham/ObjectiveC.gcda: cannot merge previous GCDA file: corrupt arc tag (0x00000000)
profiling: xx/Alcatraz/Plug-ins/Peckham/ObjectiveC.gcda: cannot merge previous GCDA file: corrupt arc tag (0x0000000a)
profiling: xx/Alcatraz/Plug-ins/Peckham/ObjectiveC.gcda: cannot merge previous run count: corrupt object tag (0x00000000)

When I remove this plugin, these message disappear.

**Comment by ClementPadovani** [comment]
Yeah this a known issue. Somehow the profiling files get corrupted.

Maybe a script to delete those files after a build would be nice. 

Thanks, Clément

**Comment by carya** [comment]
Is it means I need to write a script to delete those files after a build, this issue can not fixed within the plugin?

**Comment by ClementPadovani** [comment]
The script will be written in the plugin since the generation of those files occurs every time Alcatraz builds Peckham.
